lib/langchain/agent/base.rb

```diff
@@ -10,5 +10,10 @@ module Langchain::Agent
   #
   # @abstract
   class Base
+    def self.logger_options
```

Comment:
I think in the future we can change this to just a class method being called, a la:
```ruby
logger_class_color :red
```

something to similar to how setting description on the Tools currently works.